sirepo/template/radia.py

```diff
@@ -1694,6 +1736,43 @@ def _update_kickmap(km, und, beam_axis):
     km.periodLength = und.periodLength
 
 
+def _create_stl_trimesh(filePath):
+    trimesh.util.attach_to_log()
+    f=open(filePath)
+    mesh = trimesh.load(f, file_type='stl', force='mesh', process=True)
+    f.close()
+    return mesh
+
+
+#TODO: Necessary helper function to implement object slicing with radia.radObjMltExtPgn()
+#Edge Case: Need to remove linear points along same vecter before returning
+"""
+def sort_points_clockwise(points):
+    angles = []
+    x = [p[0] for p in points]
+    y = [p[1] for p in points]
+    center = (sum(x) / len(points), sum(y) / len(points))
+    for p in points:
+        vector = [p[0]-center[0],p[1]-center[1]]
+        vlength = math.sqrt(pow(vector[0],2) + pow(vector[1],2))
+        if vlength == 0:
+            angles.append(-np.pi)
+        else:
+            normalized = [vector[0]/vlength,vector[1]/vlength]
+            angle = math.atan2(normalized[0], normalized[1])
+            if angle < 0: #funciton checks against x-positive, if negative angle add to 2pi for mirror
```

Comment:
You can have "overline" comments. Comments in general are a [code smell](https://en.wikipedia.org/wiki/Code_smell). If you are doing something like this, make it a function that has a name instead of comments.

Reply:
I'll likely be rewriting parts of the function if we use it so I'll keep the comments and wait on seperating to another function, I appreciate the introduction to "code smell".